packages/dds/merge-tree/src/attributionCollection.ts

```diff
@@ -369,7 +385,9 @@ export class AttributionCollection implements IAttributionCollection<Attribution
 		} else {
 			this.channels ??= {};
 			if (this.channels[name] !== undefined) {
-				this.channels[name].update(undefined, channel);
+				// Non null asserting because of the check above
+				// eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+				this.channels[name]!.update(undefined, channel);
 			} else {
 				this.channels[name] = channel;
 			}
```

Comment:
Better pattern is to capture lookup into a local variable.
```suggestion
			const existingChannel = this.channels[name];
			if (existingChannel === undefined) {
				this.channels[name] = channel;
			} else {
				existingChannel.update(undefined, channel);
			}
```